scripts/lib.sh

```diff
@@ -16,13 +16,57 @@ configure-sentry-cli() {
     fi
 }
 
+# I'm assuming sw_vers is only available on Mac
```

Comment:
Reasonable assumption, but a better assertion would be `[[ $(uname -s) = 'Darwin' ]]`.

Also, you can just do this:

```sh
query-mac() {
    [[ $(uname -s) = 'Darwin' ]]
}
```

The return value of `[[` will be what the function returns as. I think it's just all `$?` in the end, but yeah.

If you want to go further than this I'd recommend just saving the output to a variable to switch case against, since subshelling is expensive.

Actually even even better, since you're using bash, bash sets `$OSTYPE`. So you could like,

```
case "$OSTYPE" in
   darwin*)
      echo mac
      ;;
   ...
esac
```
```